### DISCUSSION TITLE
Rename files with non-English characters

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [Taitava/obsidian-shellcommands#426] [Q&A]
### Obsidian debug information

Obsidian v1.6.7
Linux Mint 22
Shellcommands 0.22.0

### What shell are you using?

Bash

### Please describe the problem

Hi again, still absolutely love your plugin.

I use it to convert Obsidian files to LaTeX files to PDf files which works perfectly. I'm not actually having a bug with the plugin, but I'm struggling to create a new shell script. I've now started working with some non-English sources, that in this case include German letters ä, ö, ü, and ß. Obsidian doesn't have a problem with it, but either LaTeX or Latexmk which I use to convert the .md to .tex to .pdf does. 

I could probably troubleshoot that, but at this point I'd rather just create a little script to replace the Umlauts and ß with ae, oe, etc. 

I think {{event_title}} is what I'm looking for, in the events of `File created, File renamed, Folder created, Folder renamed`, which I've activated. 
However, I don't understand how to use this.

I'm playing around with this command now:
```
sed -e 's/ä/ae/g' -e 's/ö/oe/g' -e 's/ü/ue/g' -e 's/Ä/Ae/g' -e 's/Ö/Oe/g' -e 's/Ü/Ue/g' -e 's/ß/ss/g' {{event_title}}
```
But that's interrupting the file creation and before I can give the file a name I get this error, and always just calls the file "Untitled"

![image](https://github.com/user-attachments/assets/554cf597-1591-4148-a50d-71713e32fcd6)

Then when I rename the file afterwards I get this error:
![image](https://github.com/user-attachments/assets/eeb8710a-46fe-482b-92dc-a575b737c0f6)

Any advice appreciated!!
Thanks :)

**Comment by Taitava** [comment]
Hi! First of all, sorry for my absence!

If you still have this problem, could it be that the file is not found because the shell command is missing the `.md` file extension? To fix this, you could use [`{{event_file_name}}`](https://publish.obsidian.md/shellcommands/Variables/%7B%7Bevent_file_name%7D%7D) instead of [`{{event_title}}`](https://publish.obsidian.md/shellcommands/Variables/%7B%7Bevent_title%7D%7D), as the former variable includes file extension. So, your shell command would become:

```bash
sed -e 's/ä/ae/g' -e 's/ö/oe/g' -e 's/ü/ue/g' -e 's/Ä/Ae/g' -e 's/Ö/Oe/g' -e 's/Ü/Ue/g' -e 's/ß/ss/g' {{event_file_name}}
```

Does this help? 🙂
